reactive-streams/src/main/scala/fs2/interop/reactivestreams/StreamUnicastPublisher.scala

```diff
@@ -25,35 +25,36 @@ package reactivestreams
 
 import cats.effect.kernel._
 import cats.effect.std.Dispatcher
-import cats.syntax.all._
 
 import org.reactivestreams._
 
+import scala.util.control.NoStackTrace
+
 /** Implementation of a `org.reactivestreams.Publisher`
   *
   * This is used to publish elements from a `fs2.Stream` to a downstream reactivestreams system.
   *
+  * @note Not longer unicast, this Publisher can be reused for multiple Subscribers.
+  *
   * @see [[https://github.com/reactive-streams/reactive-streams-jvm#1-publisher-code]]
   */
-final class StreamUnicastPublisher[F[_]: Async, A] private (
+final class StreamUnicastPublisher[F[_]: Async, A](
     val stream: Stream[F, A],
-    startDispatcher: Dispatcher[F],
-    requestDispatcher: Dispatcher[F]
+    startDispatcher: Dispatcher[F]
 ) extends Publisher[A] {
-
-  @deprecated("Use StreamUnicastPublisher.apply", "3.4.0")
-  def this(stream: Stream[F, A], dispatcher: Dispatcher[F]) = this(stream, dispatcher, dispatcher)
```

Comment:
So, the new constructor is the same as the previously deprecated overload, so I just make it public to satisfy **MIMA**.
Is this correct?